docs/assets/includes/ai-info.md

```diff
@@ -0,0 +1,5 @@
+!!! note
+    -   This feature is compatible with most programming languages and requires no additional setup.
+    -   Comments are generated using the description of the static analysis issue, information about the tool that detected the issue, and a few lines of surrounding code to provide the AI with extra context and improve its accuracy.
```

Comment:
We should share how many lines we send here to address potential data-sharing concerns.

Provided we do want to share this info in the first place.